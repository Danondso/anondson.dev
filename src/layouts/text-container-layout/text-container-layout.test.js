import React from "react"
import renderer from "react-test-renderer"
import TextContainerLayout from "./text-container-layout"

describe("TextContainerLayout Snapshot test", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <TextContainerLayout>
          {" "}
          <h1>Whoah ho ho some text</h1>{" "}
        </TextContainerLayout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
