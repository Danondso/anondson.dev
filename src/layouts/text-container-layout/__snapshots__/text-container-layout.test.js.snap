// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextContainerLayout Snapshot test renders correctly 1`] = `
<section
  style={
    Object {
      "fontSize": "1.05rem",
      "maxWidth": "24rem",
      "padding": "1rem",
    }
  }
>
   
  <h1>
    Whoah ho ho some text
  </h1>
   
</section>
`;
